backend/app.py

```
@@ -0,0 +1,71 @@
+                'message': 'Failed to add to-do item'
+            }), 500
+    except Exception as e:
+        print(f"Error in submittodoitem: {str(e)}")
+        return jsonify({
+            'success': False,
+            'message': 'Internal server error'
+        }), 500
+
+if __name__ == '__main__':
+    print("Starting Flask backend server...")
+    app.run(debug=True, host='0.0.0.0', port=5001)
```

Ruff (0.13.3)

Use of `debug=True` in Flask app detected

(S201)